sdk/messaging/azservicebus/session_receiver.go

```diff
@@ -146,10 +146,17 @@ func (r *SessionReceiver) ReceiveDeferredMessages(ctx context.Context, sequenceN
 }
 
 // PeekMessages will peek messages without locking or deleting messages.
-// Messages that are peeked do not have lock tokens, so settlement methods
-// like CompleteMessage, AbandonMessage, DeferMessage or DeadLetterMessage
-// will not work with them.
+//
+// The SessionReceiver stores the last peeked sequence number internally, and will use it as the
+// start location for the next PeekMessages() call. You can override this behavior by passing an
+// explicit sequence number in PeekMessagesOptions.FromSequenceNumber.
```

Comment:
Will `[PeekMessagesOptions.FromSequenceNumber]` link to the appropriate docs?

Reply:
I actually thought it would but I had to qualify it with `azservicebus` as well.

Also, did a few more obvious ones.